test/e2e-oidc/external_oidc.go

```
@@ -0,0 +1,1239 @@
+
+func fetchIssuerJWKS(issuerURL string) (*jwks, error) {
+	client := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
+			},
+		},
+	}
+
+	// grab openid-configuration JSON which contains the URL of the provider's JWKS
+	resp, err := client.Get(issuerURL + "/.well-known/openid-configuration")
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)